articles/ros2_threat_model.md

```diff
@@ -0,0 +1,3291 @@
+---
+layout: default
+title: ROS 2 Robotic Systems Threat Model
+permalink: articles/ros2_threat_model.html
+abstract:
+  This document describes security concerns robotic systems built using ROS 2
+  may face.
+  The first section describes potential threats to ROS 2 systems while the
+  second section describes potential attacks and mitigations on a reference
+  platform (TurtleBot 3).
+author: >
+  [Thomas Moulard](https://github.com/thomas-moulard),
+  [Juan Hortala](https://github.com/juanrh)
+published: true
+categories: Security
+---
+
+{:toc}
+
+{::options parse_block_html="true" /}
+
+<style>
+table {
+    table-layout: fixed;
+    width: 100%;
+}
+
+th {
+    width: 11em;
+}
+</style>
+
+# {{ page.title }}
+
+<div class="alert alert-warning" markdown="1">
+**Disclaimer**:
+
+* This document is not exhaustive. Mitigating all attacks in this document does
+  not ensure any robotic product is secure.
+* This document is a live document. It will continue to evolve as we implement
+  and mitigate attacks against reference platforms.
+
+</div>
+
+## Document Scope
+
+This document describes potential threats for ROS 2 robotic systems. The
+document is divided into two parts:
+
+1. Robotic Systems Threats Overview
+1. Threat Analysis for the TurtleBot 3 Robotic Platform
+
+The first section lists and describes threats from a theoretical point of view.
+Explanations in this section should hold for any robot built using a
+component-oriented architecture. The second section instantiates those threats
+on a widely-available reference platform, the TurtleBot 3. Mitigating threats
+on this platform enables us to demonstrate the viability of our recommendations.
+
+## Robotic Systems Threats Overview
+
+<div class="alert alert-info" markdown="1">
+This section is intentionally independent from ROS as robotic systems
+share common threats and potential vulnerabilities. For instance, this section
+describes "robotic components" while the next section will mention "ROS 2 nodes".
+</div>
+
+### Defining Robotic Systems Threats
+
+We will consider as a robotic system one or more general-purpose computer
+connected to one or more actuators or sensors. An actuator is defined as any
+device producing physical motion. A sensor is defined as any device capturing or
+recording a physical property.
+
+### Robot Application Actors, Assets, Business Goals and Entry Points
+
+This section defines actors, assets, business goals and entry points for this
+threat model.
+
+Actors are humans or external systems interacting with the robot. Considering
```

Comment:
I had to go several times through this section. I'd suggest bolding the concept definition itself here for readability. 
```suggestion
**Actors** are humans or external systems interacting with the robot. Considering
```